src/gui/cli/Player.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <array>
+#include <iostream>
+#include "ServerData.hpp"
+
+#define INVENTORY_SIZE 7
```

Comment:
replace with
const constexpr int INVENTORY_SIZE = 7
or
const constexpr uin8_t INVENTORY_SIZE = 7
(not sure about the uint8_t, find the good one)